packages/manager/modules/enterprise-cloud-database/src/enterprise-cloud-database.controller.js

```diff
@@ -0,0 +1,56 @@
+import assign from 'lodash/assign';
+
+import { ENTERPRISE_CLOUD_DATABASE_STATUS_MAP } from './enterprise-cloud-database.constants';
+
+export default class EnterpriseCloudDatabaseCtrl {
+  /* @ngInject */
+  constructor(
+    $q,
+    $scope,
+    $state,
+    CucCloudMessage,
+    enterpriseCloudDatabaseService,
+  ) {
+    this.$q = $q;
+    this.$scope = $scope;
+    this.$state = $state;
```

Comment:
`$scope` doesn't seem used here
Also we should try to avoid injecting `$state` and `$stateParams` in the controller